### DISCUSSION TITLE
Video muito útil que não existe no momento

### DISCUSSION THREAD

**Discussion by FernandoMassa** [viniciussanchez/RESTRequest4Delphi#205] [Ideas]
Boa tarde Vinicius, vi uma resposta sobre certificado aqui (Fazer um POST enviando Certificado Digital #195) e já me serviu de guia para um trabalho que preciso fazer, mas vi seu canal no YT e vc explica muito bem e tem muita didática. Seria interessante mostrar um vídeo diferente que mostre como usar um certificado pfx. Eu trabalho com selo eletrônico e tem essa necessidade e estou tentando migrar um sistema do delphi para Lazarus e gostaria de entender mais a fundo estas funções.
Parabéns pelas tuas iniciativas e espero que goste da ideia.

**Comment by viniciussanchez** [comment]
Opa, muito obrigado... na verdade tem uma aula para ser liberado sobre isso no treinamento que fiz do RR4D. Mas vou ver mais pra frente de produzir algo no YT também...

**Comment by FernandoMassa** [comment]
Vinicius, revirei o google e não encontrei nenhum passo a passo de como instalar o Boss no linux. Vc sabe de algum?
